packages/salesforcedx-apex-replay-debugger/src/adapter/apexReplayDebug.ts

```diff
@@ -163,13 +164,63 @@ export class ScopeContainer extends VariableContainer {
   }
 }
 
+export class ApexHeapDump {
```

Comment:
Could you extract this out to its own file? I don't want `apexReplayDebug.ts` to get bloated.

Reply:
Sure. I'll create a heapDump in core since I'm pretty sure I'm going to end up needing a util file.